src/main/java/org/folio/list/services/UserFriendlyQueryService.java

```diff
@@ -18,30 +14,37 @@
 import org.folio.querytool.domain.dto.EntityDataType;
 import org.folio.querytool.domain.dto.EntityType;
 import org.folio.querytool.domain.dto.Field;
+import org.folio.spring.i18n.service.TranslationService;
 import org.springframework.stereotype.Service;
 
+import java.time.Instant;
 import java.time.LocalDate;
-import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.time.ZoneOffset;
-import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.*;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
 import java.util.function.BiFunction;
 import java.util.stream.Collectors;
 
-
 @Log4j2
 @Service
 @RequiredArgsConstructor
 public class UserFriendlyQueryService {
 
   private final EntityTypeClient entityTypeClient;
   private final FqlService fqlService;
-  private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
-  private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS");
+  private final TranslationService translationService;
+
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =  new DateTimeFormatterBuilder()
+    .append(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
+    .optionalStart().appendOffsetId() // optional Z/tz at end
+    .toFormatter().withZone(ZoneOffset.UTC); // force interpretation as UTC
```

Comment:
These are effectively equivalent, just references the builtin ISO standard one instead of our home-grown one.